### PR TITLE
DAT-15809

### PR DESCRIPTION
**Pull request by sayaliM0412**
- Consume CORE and PRO packages from GPM

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -64,6 +130,65 @@ jobs:
...
+                  "enabled": "false"
+                },
+                "snapshots": {
+                  "enabled": "true",
+                  "updatePolicy": "always"
+                }
+              },
+                {
+                  "id": "liquibase-super-pom",
+                  "url": "https://maven.pkg.github.com/liquibase/liquibase-super-pom",
```
Comment (vitaliimak) [inline .github/workflows/test.yml:163]: The project has been renamed. https://github.com/liquibase/liquibase-parent-pom

**Comment by vitaliimak** [comment]
Hi @sayaliM0412 
Need to verify that dependabot has enough rights to perform ` writing settings.xml to the path: /home/runner/.m2/settings.xml` or do any other stuff related to the `maven-settings-xml-action`.
Look at the liqiubase-commercial-mongodb repository. Dependabot PRs failed, and human PRs were successful after a similar update.

File: .github/workflows/test.yml (hunk 1 above)
Comment (sayaliM0412) [reply .github/workflows/test.yml]: Yes. Thank you. This will be taken care of in Jandro's PR. I have removed the super-pom from this PR

**Comment by jandroav** [comment]
we can remove all references to GPM regarding liquibase-parent-pom because it is now in Maven Central instead of GPM

**Comment by sayaliM0412** [comment]
> Hi @sayaliM0412 Need to verify that dependabot has enough rights to perform ` writing settings.xml to the path: /home/runner/.m2/settings.xml` or do any other stuff related to the `maven-settings-xml-action`. Look at the liqiubase-commercial-mongodb repository. Dependabot PRs failed, and human PRs were successful after a similar update.

After adding the `LIQUIBOT_PAT` to dependabot we can see [liquibase-commercial-mongodb dependabot PR is successful](https://github.com/liquibase/liquibase-commercial-mongodb/actions/runs/6197562731)

**Review by vitaliimak — APPROVED** [review]

**Pull request merged**